### PR TITLE
fix(hardcodeHero): fix hardcoded values, modify view and add filters

### PR DESCRIPTION
**Pull request by darrinmn9**

### CODE REVIEW

**Comment by tats-tagawa** [comment]
I feel you are trolling me with your colons

File: server/users/testUsers.js [hunk 1]
```diff
@@ -7,140 +7,129 @@ exports.createTestUsers = function() {
...
-        "cardNumber": 9901230120130,
-        "cvc": "123",
-        "expirationDate": '11/14',
-        "billingAddress": "Stonegate Drive",
-        "city": "San Francisco",
-        "state": "California",
-        "country": "USA",
-      });
+    .then(function(user) {
+      if (user) {} else {
```
Comment (tats-tagawa) [inline server/users/testUsers.js:11]: I wrote the original code and I want to stab myself for writing such terrible code. That said, can't we just write `if (!user) { ... create user ... }`?

Comment (darrinmn9) [inline server/users/testUsers.js:11]: yes

Comment (darrinmn9) [inline server/users/testUsers.js:11]: sorry about the : i just cant help it

**Pull request merged**